<launch>
  <node pkg="ca_mpu6050" type="mpu6050_node.py" name="mpu6050">
    <rosparam command="load" file="$(find ca_mpu6050)/config/config.yaml"/>
    <rosparam command="load" file="$(find ca_mpu6050)/config/calibrated.yaml"/>
    <param name="~debug" value="true"/>
  </node>
</launch>
